docs/openj9_support.md

```diff
@@ -79,27 +79,33 @@ platform support might vary, depending on their build environment.
 
 OpenJDK8 binaries are supported on the minimum operating system levels shown in the following tables:
 
-
-| Linux                                 |   x32  |  x64   | ppc64le    | Z31  | Z64 |
+| Linux&reg;                            |   x32  |  x64   | ppc64le    | Z31  | Z64 |
 |---------------------------------------|--------|--------|------------|------|------|
-| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
-| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
-| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
-| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| Centos 6.9                            |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7.4                            |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6.9   |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7.4                              |   Y    |   Y    |     Y      |   Y  |  Y   |
 | SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
 | Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
 | Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
 
 <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Not all of these distributions are tested, but Linux distributions that have a
-minimum glibc version 2.12 are expected to function without problems.
+minimum glibc version 2.12 on x86 or glib version 2.17 on other architectures are expected to function without problems.
```

Comment:
I don't want this statement change to come back and cause problems. zlinux (and plinux BE) are still supported on 2.12 by IBM Java 8, which uses OpenJ9. I expect this means OpenJ9 will continue to be supported on 2.12 for these platforms, regardless of the minimum OSes Adopt supports.

Reply:
Fair point. I'll change it back.